packages/block/src/header-from-rpc.ts

```diff
@@ -1,15 +1,18 @@
 import { BlockHeader } from './header'
 import { numberToHex } from './helpers'
 
-import type { BlockOptions } from './types'
+import type { BlockOptions, JsonRpcBlock } from './types'
 
 /**
  * Creates a new block header object from Ethereum JSON RPC.
  *
  * @param blockParams - Ethereum JSON RPC of block (eth_getBlockByNumber)
  * @param options - An object describing the blockchain
  */
-export function blockHeaderFromRpc(blockParams: any, options?: BlockOptions) {
+export function blockHeaderFromRpc(
+  blockParams: Omit<JsonRpcBlock, 'receiptsRoot'> & { receiptRoot?: string; receiptsRoot?: string },
```

Comment:
Do we still need this `receiptRoot` in here?  I thought we decided to do away with it in #2256?  I looked at the Ethereum JSON RPC specs and it's definitely `receiptsRoot` in the [JSON-RPC definition](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getblockbyhash)

Reply:
oh no we don't, messed that up during the "extraction" of the large PR into package-specific PRs, fixed